bloom-housing/bloom
# feat: toggle actions - listings approval

# Pull Request Template

## Issue Overview

This PR addresses [#307](https://github.com/metrotranscom/doorway/issues/307)


## Description

This PR just adds the status changes for listings approval in between actions in the flow.

## How Can This Be Tested/Reviewed?

You'll need to update the new partner's env variable to true.

You can see the intended statuses at each step [here](https://docs.google.com/document/d/1v-HgqnIyI9XtHmmDeHpXqbM1lS7FbJDjbu4GBdLj6kU/edit?usp=sharing).

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
